content/tutorials/mushr_navigation_system.md

```diff
@@ -0,0 +1,164 @@
+Make sure you’re using the sandbox map. The quickstart tutorial explains how to change the map.
+
+
+## Environment setup
+Clone the [nhttc_ros](https://github.com/naughtyStark/nhttc_ros.git) repository into your catkin workspace: 
+{{<highlight bash>}}
+$ cd catkin_ws/src
+$ git clone --branch devel https://github.com/naughtyStark/nhttc_ros.git
+$ cd nhttc_ros
+$ git submodule init
+$ git submodule update --recursive
+{{</highlight>}}
+
+Install python requirements:
+{{<highlight bash>}}
+$ cd ~/catkin_ws/src/nhttc_ros
+$ pip install -r requirements.txt
+{{</highlight>}}
+
+Compile using catkin_make:
+{{<highlight bash>}}
+$ cd ~/catkin_ws
+$ catkin_make
+{{</highlight>}}
+
+
+## Running the example:
+After everything compiles, you should be ready to try out the simulation example. Launch the nhttc_demo.launch with the following command:
+{{< highlight bash >}}
+$ roslaunch nhttc_ros nhttc_demo.launch
+{{< / highlight >}}
+
+Wait till you see the following text on the terminal (you may see a few extra lines):
+``` bash
+[ INFO] [1621678112.116392844]: node started
+[ INFO] [1621678112.116614789]: carrot_goal_ratio: 1.000000
+[ INFO] [1621678112.116632550]: max_ttc: 6.000000
+[ INFO] [1621678112.116642780]: solver_time: 20
+[ INFO] [1621678112.116653036]: obey_time:1
+```
+
+In a new tab of the terminal (use Ctrl + Shift + T), open rviz:
+{{<highlight bash>}}
+$ rviz -d ~/catkin_ws/src/nhttc_ros/nhttc_ros/rviz/nhttc.rviz
+{{</highlight>}}
+
+In another new tab, run the route publisher script:
+{{<highlight bash>}}
+$ rosrun nhttc_ros route_publisher.py
+{{</highlight>}}
+
+You should see something like this on rviz:
+<br>
+{{< figure src="/tutorials/MuSHR_multiagent_navigation/demo_default.gif" width="800" >}}                           
+
+This should be considered the default case henceforth. In this demo, the blue car is trying to follow the blue arrows and the green car is trying to follow the green arrows. The green car is supposed to stop a small distance after crossing the intersection of the two paths whereas the blue car is supposed to move towards the top after the intersection of the two paths. The two paths coincide to force the navigation system to display its capabilities. **Note that you may observe slightly different behaviour when you launch this on your system, the provided gif is only for reference.**
+
+**Note:** In order to rerun this example you must close all the processes we started here and follow the steps from the beginning of this section.
+
+
+## Using the wrapper for your own intents and purposes:
+The system takes a set of waypoints rather than a single goal point. The message to publish for this is:`/car_name/waypoints` of type: [geometry_msgs/PoseArray](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseArray.html). Note that "car_name" refers to the name of the car, such as car1, car2, and so on.
```

Comment:
Could they just give a single goal point in the PoseArray if they don't have a planned path?

Reply:
Seems like you should mention editing route_publisher as a easy way to change waypoints....